account/manager.go

```
@@ -908,7 +898,7 @@ func (m *manager) deriveFeeFromTx(ctx context.Context,
 	for _, txIn := range tx.TxIn {
 		spendTx, err := m.locateTxByHash(ctx, txIn.PreviousOutPoint.Hash)
 		if err != nil {
-			return 0, nil
+			return nil, nil
```

Oh, I guess this slipped by in my last review. This should return the error.

---

Good catch. Thanks.